projects.json

```
@@ -2036,6 +2081,66 @@
       }
     ]
   },
+  {
+    "repository": "Git",
+    "url": "https://github.com/DeclarativeHub/ReactiveKit.git",
+    "path": "ReactiveKit",
+    "branch": "master",
+    "compatibility": [
+      {
+        "version": "4.2",
```

Update compatibility version to 5.1
```suggestion
        "version": "5.1",
```